conf/messages

```diff
@@ -3,4 +3,7 @@ invalid.action=Review the application
 override.scopes.required=Scopes are required when enabling this access override
 override.scopes.incorrect=Error: You must choose at least one correct scope
 scopes.required=Scopes are required
-invalid.scope=You have used a scope we don't recognise
\ No newline at end of file
+invalid.scope=You have used a scope we don't recognise
+application.administrator.missing=Choose an administrator
+application.confirmation.missing=Provide the application name
+application.confirmation.error=The application name doesn''t match
```

Comment:
Does this need the double single quote? The `invalid.scope` message doesn't have that